docs/usage.rst

```diff
@@ -110,18 +110,18 @@ Nengo's scripting environment as well:
 Examples
 ========
 
-NengoFPGA ships with a few example implementations in the `nengo-fpga/docs/examples`
+NengoFPGA ships with a few example implementations in the ``nengo-fpga/docs/examples``
 folder. These examples are designed to be used with Nengo GUI, so first we will
 install that.
 
 1. Install the GUI with ``pip install nengo-gui``.
-#. In a terminal window, navigate to the `nengo-fpga/docs/examples` directory.
+#. In a terminal window, navigate to the ``nengo-fpga/docs/examples`` directory.
 #. Try running an example with ``nengo <file name> -b nengo_fpga``. This
    should open the Nengo GUI interface in a browser and display the code on the
    right and a graphical representation on the left.
-#. Near the top of the file you should see `# --- BOARD SELECT ---`, select
-   the appropriate board here. (The names `de1` and `pynq` correspond to the
-   headers in the `fpga_config` file).
+#. Near the top of the file you should see ``# --- BOARD SELECT ---``, select
```

Comment:
we should standardize this. In the getting started section, contents in files are bolded, here, they are in code blocks? I'm not sure which is best to do though.

Reply:
I guess, do the one that requires the least work? haha

Reply:
I intended to do backticks for files/paths/code and use bold for names and things. I probably just left these as code snippets since they *are* in the code, but I think it makes more sense to make them bold (like I did for the entries of `fpga_config` which you allude to).